### PR TITLE
KGQA skill

### PR DESCRIPTION
**Pull request by HaishuoFang**
# What does this PR do?

### KGQA skill
- model: a seq2seq BART model 
- Dataset: KQApro dataset
- Workflow:
    - Get the generated SPARQL from the model API.
    - Postprocess the generated SPARQL.
    - Query this SPARQL against KB from Virtuoso.
    - Get the answer. 

## Who can review?

Feel free to tag members/contributors who may be interested in your PR.

@HaritzPuerto @Rachneet @timbmg

### CODE REVIEW

**Comment by timbmg** [comment]
Currently, we only copy the `skill.py` file into the image. It seems your skill requires some more files. Would it be possible to have it all in one file? Or else, you would need to change the [Dockerfile](https://github.com/UKP-SQuARE/square-core/blob/9eb74d69618c069518cba8d35c879f59cce7e2bc/skills/Dockerfile), particularly, [this line](https://github.com/UKP-SQuARE/square-core/blob/9eb74d69618c069518cba8d35c879f59cce7e2bc/skills/Dockerfile#L17)

File: skills/KQAPro-KGQA/readme.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+## Package
```
Comment (timbmg) [inline skills/KQAPro-KGQA/readme.md:1]: Is this file required to check-in to the main repo? Seems just like your development comments?:)

**Review by timbmg — CHANGES_REQUESTED** [review]

File: config.yaml [hunk 2]
```diff
@@ -27,6 +27,7 @@ skills:
   - name: extractive-metaqa
     author: ukp
   - name: multiple-choice-metaqa
+  - name: KQAPro-KGQA
```
[the remark this answers is not in the document]
Comment (HaishuoFang) [reply config.yaml:30]: I already added it. `multiple-choice-metaqa` needs to add one i think. I will add it

**Review by timbmg — APPROVED** [review]

**Comment by timbmg** [comment]
LGTM. If you like you can deploy to prod and check if it works as expected.

**Pull request merged**